### DISCUSSION TITLE
Redirect a .gitlab.io page to .vercel.app

### DISCUSSION THREAD

**Discussion by reveurguy** [vercel/community#1389] [Help] [answered]
### Summary

I am working on a project which is deployed using gitlab page. I have created a vercel deployment for the project and want to redirect the .gitlab.io to my vercel deployment automatically.
So whenever someone opens the gitlab link, automatically the vercel deployment opens.

For that i added a custom domain in Gitlab page as name.vercel.app and copied the ALIAS record and verification TXT mentioned in the custom domain setting to a domain in vercel but it is not working.

How can i make it work, what steps would i need to follow if i am doing anything wrong above!

Thanks

**Comment by MrAmericanMike** [comment, marked as answer]
I believe you could ask on Gitlab for more details. But check this page.

https://docs.gitlab.com/ee/user/project/pages/redirects.html

They mention that `Domain-level redirects` are not supported (At least with that method) but maybe there are others. I don't think from what I have seen that Gitlab supports `.htaccess` neither.
